index.html

```diff
@@ -43,6 +43,7 @@
         <li><a href="/user/build-timeouts">My builds are timing out, what can I do?</a></li>
         <li><a href="/user/repository-providers">What repository providers can I use?</a></li>
         <li><a href="/user/pricing/">Why can't I find pricing information?</a></li>
+        <li><a href="/user/ci-environment-trusty-beta/">Can I use Ubuntu Trusty for my builds?"</a><li>
```

Comment:
I'm in favor of dropping `beta`.  Is there any prior art?  Why do we even have a `/user` namespace anyway? :-P